x/ccv/provider/types/msg.go

```diff
@@ -746,112 +763,133 @@ func ValidateField(nameOfTheField string, field string, maxLength int) error {
 	return nil
 }
 
+// TruncateString truncates a string to maximum length characters
+func TruncateString(str string, length int) string {
```

Comment:
s/length/maxLength

Why not getting the minimum `m` of `len(str)` and `maxLenght` and then `return str[:m]` instead?

Reply:
Tend to agree, this is a one-liner that can be inlined in funcs.

Reply:
Used the solution [here](https://dev.to/takakd/go-safe-truncate-string-9h0) in case we want to have Chains with Japanese description :)